### DISCUSSION TITLE
менеджер стоп/запуск

### DISCUSSION THREAD

**Discussion by Eirev** [fedorovvl/tso_client#19] [Ideas]
идея в связи с появлением большого числа специальных строений для бафа площади, позволяющих регулярно бафать области

дело в том что под той же ярмаркой может умещаться до 30 зданий (может кто-то умещал и больше). кому-то может потребоваться резко поставить все на паузу и после запустить. аналогично с деревом, которое работает по 6 часов и регулярно надо под ним останавливать и запускать строения.

можно ли сделать менеджер для запуска и остановки зданий по аналогии с менеджером бафов?

то есть запускаю запись, останавливаю производство, сохраняю.
после запускаю запись, запускаю производство, сохраняю.

если это сложно или нужно только мне, тогда идея снимается.

**Comment by BasilMas** [comment]
для деревьев/ярмарок/етс, пожалуй, и я не откажусь от такой фичи.

**Reply by fedorovvl** [reply]
может автопрохождение приклов по шаблонам? куда то вас уже понесло помоему..

**Comment by BasilMas** [comment]
Нет, идея вот в чем - баф по области то же дерево, к примеру, это та же рутина старт/стоп определенного числа зданий по времени. У меня под деревом примерно 35 зданий, у кого-то под ярмаркой и того больше. Всегда есть шанс что-то промазать стартануть/стопнуть. Т.е. по сути это аналог бафа, только вместо накладывания бафа старт/стоп здания.
Ну опять же - нет так нет)

**Comment by Eirev** [comment]
хочу сразу сказать, выражаю огромную благодарность вам за всю проведенную работу.
со времени перехода на юнити никто не сделал для игры и ее сообщества больше, чем вы.

я прекрасно понимаю, что вы занимаетесь этим в свое свободное время и по своему желанию.
поэтому не рассматривайте мои идеи как просьбы или даже пожелания. это просто идеи

возможно сейчас у вас нет времени и желания заниматься чем-то подобным, но ведь вы же сделали менеджер спецов, вдруг и еще раз звезды сойдутся?

может это произойдет через три месяца. или через год. или никогда :)
моя благодарность к вам от этого не изменится, я все равно буду вам благодарен за ваш труд

что касается самой идеи, прохождение приключений это справедливая игровая механика.
отключение/включение нескольких десятков зданий это рутинная механика, возникшая стихийно.
она не несет никакого смысла, игрок просто должен тыкать в каждое здание каждый раз когда баф спал или активировался.
теперь, когда это можно делать регулярно на большом количестве построек, это серьезно надоедает.

все же это не запрос, а идея.

**Comment by fedorovvl** [comment]
реализовано в 1.5.3
